File: archaic/src/main/scala/resonantinduction/archaic/waila/WailaRegistrar.java
package resonantinduction.archaic.waila;

import mcp.mobius.waila.api.IWailaRegistrar;
import resonantinduction.archaic.crate.TileCrate;
import resonantinduction.archaic.fluid.tank.TileTank;

public class WailaRegistrar
{
    public static void wailaCallBack(IWailaRegistrar registrar)
    {
        registrar.registerBodyProvider(new WailaCrate(), TileCrate.class);
        registrar.registerBodyProvider(new WailaFluidTank(), TileTank.class);
    }
}

File: src/main/scala/edx/basic/waila/WailaRegistrar.scala
package edx.basic.waila

import edx.basic.fluid.tank.TileTank
import mcp.mobius.waila.api.IWailaRegistrar

object WailaRegistrar
{
  def wailaCallBack(registrar: IWailaRegistrar)
  {
    registrar.registerBodyProvider(new WailaFluidTank, classOf[TileTank])
  }
}